packages/traceability-schemas/schemas/Event.json

```diff
@@ -0,0 +1,67 @@
+{
+    "$id": "https://w3id.org/traceability/schemas/Event.json",
+    "$schema": "https://json-schema.org/draft-07/schema#",
+    "$comment": "{\"term\": \"Event\", \"@id\": \"https://schema.org/Event\"}",
+    "title": "Organization",
+    "description": "An event such as transformation, aggregation, commission etc.",
+    "type": "object",
+    "properties": {
+        "@context": {
+            "type": "array"
+        },
+        "type": {
+            "oneOf": [
+                {
+                    "type": "string"
+                },
+                {
+                    "type": "array"
+                }
+            ]
+        },
+        "eventType": {
+            "$comment": "{\"term\": \"eventType\", \"@id\": \"https://schema.org/value\"}",
+            "title": "Event Type",
+            "description": "Type of Event",
+            "type": "string"
+        },
+        "eventId": {
+            "$comment": "{\"term\": \"eventId\", \"@id\": \"https://schema.org/identifier\"}",
+            "title": "Event Identifier",
+            "description": "Event Identifier.",
+            "type": "string"
+        },
+        "actor": {
+            "$comment": "{\"term\": \"actor\", \"@id\": \"https://w3id.org/traceability#Organization\", \"@type\": \"https://schema.org/ItemList\"}",
+            "title": "Actor",
+            "description": "The organization performing the activity.",
+            "type": "array",
+            "items": {
+                "$ref": "https://w3id.org/traceability/schemas/Organization.json"
+            }
+        },
+        "eventLocation": {
+            "$comment": "{\"term\": \"eventLocation\", \"@id\": \"https://w3id.org/traceability#eventLocation\"}",
```

Comment:
I am not sure we want to be defining "eventLocation"... can we use an existing ontology for this field?

Reply:
+1 I think this should be a [Place](https://w3c-ccg.github.io/traceability-vocab/#Place)

Reply:
absolutely should be place